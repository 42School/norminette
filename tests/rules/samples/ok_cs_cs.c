int	main(void)
{
	while (1)
		return ;
	break ;
}
