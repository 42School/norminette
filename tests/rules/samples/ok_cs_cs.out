[36mok_cs_cs.c[0m - [32mIsFuncDeclaration[0m In "GlobalScope" from "None" line 1":
		<INT> <TAB> <IDENTIFIER=main> <LPARENTHESIS> <VOID> <RPARENTHESIS> <NEWLINE>
[36mok_cs_cs.c[0m - [32mIsBlockStart[0m In "Function" from "GlobalScope" line 2":
		<LBRACE> <NEWLINE>
[36mok_cs_cs.c[0m - [32mIsControlStatement[0m In "Function" from "GlobalScope" line 3":
		<TAB> <WHILE> <SPACE> <LPARENTHESIS> <CONSTANT=1> <RPARENTHESIS> <NEWLINE>
[36mok_cs_cs.c[0m - [32mIsExpressionStatement[0m In "ControlStructure" from "Function" line 4":
		<TAB> <TAB> <RETURN> <SPACE> <SEMI_COLON> <NEWLINE>
[36mok_cs_cs.c[0m - [32mIsExpressionStatement[0m In "Function" from "GlobalScope" line 5":
		<TAB> <BREAK> <SPACE> <SEMI_COLON> <NEWLINE>
[36mok_cs_cs.c[0m - [32mIsBlockEnd[0m In "Function" from "GlobalScope" line 6":
		<RBRACE> <NEWLINE>
ok_cs_cs.c: Error!
Error: INVALID_HEADER       (line:   1, col:   1):	Missing or invalid 42 header
